test/converters/converters.go

```diff
@@ -0,0 +1,21 @@
+package converters
+
+import (
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/junitxml"
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/xcresult"
+)
+
+type Intf interface {
```

Comment:
what about `Converter`

Reply:
converters.Converter?

Reply:
also, the converter is that matching to the interface

Reply:
The interface methods reduced to XML() and Detect(). I think now both of these describe the converter properly. XML() is a Go convention for typed outputs like String() etc..

Reply:
There is a converter interface to match for each converter. I will remove Handler completely as it is pointed out it sounds misleading.